pkgs/top-level/perl-packages.nix

```diff
@@ -13871,6 +13871,24 @@ let
      };
   };
 
+  NetServerSSPrefork = buildPerlPackage {
+     pname = "Net-Server-SS-PreFork";
+     version = "5fccc0c270";
```

Comment:
maybe `0.06pre` or `0.05.2017-06-08` ?
to be newer than CPAN's `0.05` (https://metacpan.org/pod/Net::Server::SS::PreFork) so the updated won't go mad